.github/workflows/help-command.yml

```diff
@@ -0,0 +1,23 @@
+name: help-command
+
+on:
+  repository_dispatch:
+    types: [help-command]
+
+jobs:
+  help:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update comment
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          token: ${{ secrets.NEPTUNE_BOT_ACCESS_TOKEN }}
+          comment-id: ${{ github.event.client_payload.github.payload.comment.id }}
+          body: |
+            > Command | Description
+            > --- | ---
+            > /pre-commit-apply | Commit a pre-commit suggestions
+            > /run-unit | Run unittests on full matrix
+            > /run-e2e | Run E2E tests on full matrix
```

Comment:
`/run-all-test` = `run-e2e + run-integrations` ?

Reply:
Sure :wink: as you wish